app/services/decisions/income_decision_tree.rb

```diff
@@ -10,6 +10,8 @@ def destination
         edit(:manage_without_income)
       when :income_before_tax
         after_income_before_tax
+      when :income_savings_assets
```

Comment:
Does this name make sense to you?

Reply:
Are there going to be other screens that ask about income, savings and assets? If so then perhaps it's too vague and should be similar to the method below `has_frozen_income_or_assets` so maybe `frozen_income_savings_assets` and `after_frozen_income_savings_assets`

Reply:
It's hard to represent such a long question without making the step name too long though so i'm not fussed 😄